po/Makefile.am

```diff
@@ -125,7 +125,7 @@ upload-translations:
 
 download-po:
 	$(ZANATA_CLI) pull --includes=cockpit.pot --project-config "$(srcdir)/po/zanata.xml" \
-		--min-doc-percent 1 -s "$(builddir)/po" -t "$(srcdir)/po"
+		--min-doc-percent 50 -s "$(builddir)/po" -t "$(srcdir)/po"
```

Comment:
Can you please add https://bugzilla.redhat.com/show_bug.cgi?id=1666722 to the commit message?